src/app/shared/components/template/components/latex/latex.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from "@angular/core";
+import { TemplateBaseComponent } from "src/app/shared/components/template/components/base";
+
+@Component({
+  selector: "template-latex-component",
+  template: ` <span [innerHTML]="latexInput | latex"> </span> `,
+})
+/** Render raw LaTeX **/
+export class TmplLatexComponent extends TemplateBaseComponent implements OnInit {
+  latexInput: string;
+
+  constructor() {
+    super();
```

Comment:
You only need to make calls to the `super` class if using the constructor (e.g. injecting another service). So you can simply omit here